src/Vegetation/PlantHydraulics.jl

```diff
@@ -431,28 +439,30 @@ function flux_out_roots(
     p::ClimaCore.Fields.FieldVector,
     t::FT,
 )::FT where {FT}
-    @unpack vg_α, vg_n, vg_m, ν, S_s, K_sat = model.parameters
-
-    S_l_stem = effective_saturation(ν, Y.vegetation.ϑ_l[1])
-
-    #How to ungeneralize
-
-    p_stem = water_retention_curve.(vg_α, vg_n, vg_m, S_l_stem, ν, S_s)
-    return sum(
+    @unpack plant_vg_α, plant_vg_n, plant_vg_m, plant_ν, plant_S_s, above_ground_plant_K_sat, K_sat_root, root_distribution = model.parameters
+    S_l_stem = effective_saturation(plant_ν, Y.vegetation.ϑ_l[1])
+    # How to ungeneralize
```

Comment:
rm comment

Reply:
Ok, what was meant by this?

Reply:
oh, just no need to merge in "#How to ungeneralize" into main :)

Comments do have a place, but good to clean up little notes to yourself that arent relevant anymore before merging

Reply:
cool :-)